packages/hardhat/scripts/deploy.ts

```diff
@@ -20,6 +20,8 @@ import {
   SVGPlaceholder__factory,
 } from "../typechain";
 
+const SERVICE_FEE_ADDRESS = "0x4cce31b5E734934c62f0F411BaF53c2b3e14c3f4"; // TODO update this to DAO address
```

Comment:
curious - what's the meaning of this address?

Reply:
@eladmallel just the address I use for testnet